README.rst

```diff
@@ -555,7 +556,7 @@ Choose from the list of available rules:
    | *Rule is: risky.*
 
 * **simplified_null_return**
-   | A return statement wishing to return nothing should be simply "return".
+   | A return statement wishing to return "void" should not return "null".
```

Comment:
>and the new one: [...] is suggesting that we do check the willingness

yes, like the old one, so 

>maybe just don't modify the description

doesn't solve it.

Going back and forth over copy makes no sense to me, just tell me what to put in.
Same with:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2339#discussion_r90223642
You don't want the defaults set to `null`, I don't want to pass `null` everywhere, the result is the tests breaking. So what to do with this PR, its getting time consuming without much process.